[2023-05-31T11:12:06.463+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: birthday_email_dag.send_birthday_emails scheduled__2023-05-29T00:00:00+00:00 [queued]>
[2023-05-31T11:12:06.475+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: birthday_email_dag.send_birthday_emails scheduled__2023-05-29T00:00:00+00:00 [queued]>
[2023-05-31T11:12:06.475+0000] {taskinstance.py:1308} INFO - Starting attempt 24 of 24
[2023-05-31T11:12:06.509+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): send_birthday_emails> on 2023-05-29 00:00:00+00:00
[2023-05-31T11:12:06.514+0000] {standard_task_runner.py:57} INFO - Started process 13053 to run task
[2023-05-31T11:12:06.518+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'birthday_email_dag', 'send_birthday_emails', 'scheduled__2023-05-29T00:00:00+00:00', '--job-id', '253', '--raw', '--subdir', 'DAGS_FOLDER/birthday_dag.py', '--cfg-path', '/tmp/tmprjfxi4ld']
[2023-05-31T11:12:06.520+0000] {standard_task_runner.py:85} INFO - Job 253: Subtask send_birthday_emails
[2023-05-31T11:12:06.595+0000] {task_command.py:410} INFO - Running <TaskInstance: birthday_email_dag.send_birthday_emails scheduled__2023-05-29T00:00:00+00:00 [running]> on host ce3f9f28322a
[2023-05-31T11:12:06.780+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='birthday_email_dag' AIRFLOW_CTX_TASK_ID='send_birthday_emails' AIRFLOW_CTX_EXECUTION_DATE='2023-05-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='24' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-29T00:00:00+00:00'
[2023-05-31T11:12:06.782+0000] {sql.py:262} INFO - Executing: SELECT employee_email, employee_name FROM employee WHERE DATE_FORMAT(employee_date_of_birth, '%m-%d') = DATE_FORMAT(NOW(), '%m-%d')
[2023-05-31T11:12:06.797+0000] {base.py:73} INFO - Using connection ID 'hrms' for task execution.
[2023-05-31T11:12:06.912+0000] {base.py:73} INFO - Using connection ID 'hrms' for task execution.
[2023-05-31T11:12:06.916+0000] {sql.py:375} INFO - Running statement: SELECT employee_email, employee_name FROM employee WHERE DATE_FORMAT(employee_date_of_birth, '%m-%d') = DATE_FORMAT(NOW(), '%m-%d'), parameters: None
[2023-05-31T11:12:06.917+0000] {sql.py:384} INFO - Rows affected: 6
[2023-05-31T11:12:06.919+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/birthday_dag.py", line 70, in send_birthday_emails
    server = smtplib.SMTP(smtp_host, smtp_port)
  File "/usr/local/lib/python3.7/smtplib.py", line 251, in __init__
    (code, msg) = self.connect(host, port)
  File "/usr/local/lib/python3.7/smtplib.py", line 336, in connect
    self.sock = self._get_socket(host, port, self.timeout)
  File "/usr/local/lib/python3.7/smtplib.py", line 307, in _get_socket
    self.source_address)
  File "/usr/local/lib/python3.7/socket.py", line 707, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/usr/local/lib/python3.7/socket.py", line 752, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -3] Temporary failure in name resolution
[2023-05-31T11:12:06.933+0000] {taskinstance.py:1350} INFO - Marking task as FAILED. dag_id=birthday_email_dag, task_id=send_birthday_emails, execution_date=20230529T000000, start_date=20230531T111206, end_date=20230531T111206
[2023-05-31T11:12:06.955+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 253 for task send_birthday_emails ([Errno -3] Temporary failure in name resolution; 13053)
[2023-05-31T11:12:06.971+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-05-31T11:12:06.997+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
